services/web/src/screens/Organizations/Actions.js

```diff
@@ -1,35 +1,111 @@
-import { Dropdown } from 'semantic';
-
-import InspectObject from 'modals/InspectObject';
-import Confirm from 'components/Confirm';
+import { ActionIcon, Menu, Text, Button, Group } from '@mantine/core';
+import { modals } from '@mantine/modals';
+import {
+  IconTrash,
+  IconCode,
+  IconListSearch,
+  IconDotsVertical,
+  IconPencil,
+  IconEdit,
+} from '@tabler/icons-react';
+import { Link } from '@bedrockio/router';
 
+import InspectObject from 'components/InspectObject';
+import ConfirmModal from 'components/ConfirmModal';
+import Protected from 'components/Protected';
 import { request } from 'utils/api';
 
-export default function OrganizationActions(props) {
-  const { organization, reload } = props;
-  return (
-    <Dropdown button basic text="More">
-      <Dropdown.Menu direction="left">
-        <InspectObject
-          name="Organization"
-          object={organization}
-          trigger={<Dropdown.Item text="Inspect" icon="code" />}
-        />
-        <Confirm
+export default function OrganizationActions({ compact, organization, reload }) {
+  function openInspectModal() {
+    modals.open({
+      title: `Inspect ${organization.name}`,
+      children: <InspectObject object={organization} name="organization" />,
+      size: 'lg',
+    });
+  }
+
+  function openDeleteModel() {
+    modals.open({
+      title: `Delete Organization`,
+      children: (
+        <ConfirmModal
           negative
-          confirmButton="Delete"
-          header={`Are you sure you want to delete "${organization.name}"?`}
-          content="All data will be permanently deleted"
-          trigger={<Dropdown.Item text="Delete" icon="trash" />}
           onConfirm={async () => {
             await request({
               method: 'DELETE',
               path: `/1/organizations/${organization.id}`,
             });
-            reload();
+            return reload();
```

Comment:
Why is this returning? This seems odd

Reply:
I was thinking that reload was a promise, but its not, so will kill it.